CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Since there are no consistency guarantees provided by some Object Storage provid
 - [#2294](https://github.com/thanos-io/thanos/pull/2294) store: optimizations for fetching postings. Queries using `=~".*"` matchers or negation matchers (`!=...` or `!~...`) benefit the most.
 - [#2301](https://github.com/thanos-io/thanos/pull/2301) Ruler: initlialization fails with filepath bad pattern error and rule manager update error.
 - [#2310](https://github.com/thanos-io/thanos/pull/2310) query: Report timespan 0 to 0 when discovering no stores.
+- [#2330](https://github.com/thanos-io/thanos/pull/2330) store: index-header is no longer experimental. It is enabled by default for store Gateway. You can disable it with new hidden flag: `--store.disable-index-header`. `--experimental.enable-index-header` flag was removed.
```

Comment:
```suggestion
- [#2330](https://github.com/thanos-io/thanos/pull/2330) store: index-header is no longer experimental. It is enabled by default for store Gateway. You can disable it with new hidden flag: `--store.disable-index-header`. Important! `--experimental.enable-index-header` flag was removed.
```

Reply:
We need to somehow msg this loud as it will break people if they don't update ;)

Reply:
Yes, but it's just an hidden experimental flag. It was obvious it will change. So no braking change within our guidelines.

Reply:
:+1: